datalad_tabby/conftest.py

```diff
@@ -1,6 +1,6 @@
 from datalad.conftest import setup_package
 
-from datalad_next.tests.fixtures import datalad_noninteractive_ui
+pytest_plugins = "datalad_next.tests.fixtures"
```

Comment:
At first I didn't understand where the `existing_dataset` fixture was coming from but then I found out about the `pytest_plugins` syntax that I wasn't aware of before. Neat.